ac-dcd.el

```diff
@@ -537,8 +537,8 @@ dcd-client outputs candidates that begin with \"this\" when completing struct co
     (with-current-buffer buf
       (erase-buffer)
 
-          (apply 'call-process-region (point-min) (point-max)
```

Comment:
?

Reply:
Correction of indentation that sneaked in without me noticing it.